src/js/ctrl/signals.ts

```diff
@@ -66,7 +62,7 @@ api.isDefault = isDefault;
  * @returns {*} The value of the signal
  */
 api.get = function(name) {
-  var ctrl = require('../'),
+  var ctrl = require('./'),
```

Comment:
Should probably refactor to use const/let instead of var while you're upgrading to typescript.

Reply:
Though if you're planning on rewriting this file anyways, I guess that's okay - I'll give the approval since everything else looks good!

Reply:
Although wait this line itself is really weird? Why are you requiring yourself?

Reply:
yeah this folder was kinda wacky -- it's requiring the index in that directory